addons/website/static/src/snippets/s_table_of_content/000.scss

```diff
@@ -0,0 +1,64 @@
+.s_table_of_content:not([data-vcss]) {
+    .s_table_of_content_navbar_wrap {
+        &.s_table_of_content_navbar_sticky {
+            &.s_table_of_content_horizontal_navbar, &.s_table_of_content_vertical_navbar .s_table_of_content_navbar {
+                @include o-position-sticky($top: 0px);
+            }
+        }
+        &:not(.s_table_of_content_navbar_sticky) {
+            &, .s_table_of_content_navbar {
+                top: 0px !important;
```

Comment:
Why important ?

Reply:
because the top position of the sticky navbar is defined in js..
so the only solution to avoid this important would be to use a custom method instead of data-select-class to change the top property when it's not sticky

Reply:
No, it's ok :+1: